applications/nrf_desktop/configuration/common/led_effect.h

```diff
@@ -116,6 +117,35 @@ struct led_effect {
 		.loop_forever = true,						\
 	}
 
+
```

Comment:
explain in the comment here why we cannot give period as argument

Reply:
done